src/js/libgeo/GeoOps.js

```diff
@@ -713,6 +755,26 @@ geoOps.CircleMr.updatePosition = function(el) {
     el.matrix = General.withUsage(matrix, "Circle");
     el.radius = r;
 };
+geoOps.CircleMr.getRandomMove = function(el) {
+    // radius 
+    var r;
+    var oldr = el.radius;
+    var oabs = CSNumber.abs(oldr).value.real;
+
+    // if radius was small we want something larger and if not we scale the old one
+    if (oabs < CSNumber.eps) {
+        r = CSNumber.getRandReal(0.05, 0.10);
+    } else {
+        r = CSNumber.mult(oldr, CSNumber.getRandReal(0.95, 1.05));
+    }
```

Comment:
Is there a good reason why the radius has to be real? If there is, it should be commented upon. If not, choosing a complex radius would be more consistent.